smoketest/scripts/cli/base_interfaces_test.py

```diff
@@ -278,18 +278,119 @@ def test_vif_8021q_interfaces(self):
                     self.cli_set(base + ['mtu', self._mtu])
                     for address in self._test_addr:
                         self.cli_set(base + ['address', address])
+                    self.cli_set(base + ['ingress-qos', '0:1'])
+                    self.cli_set(base + ['egress-qos', '1:6'])
 
             self.cli_commit()
 
             for intf in self._interfaces:
                 for vlan in self._vlan_range:
                     vif = f'{intf}.{vlan}'
+                    tmp = get_interface_config(f'{vif}')
+                    
+                    tmp2 = dict_search('linkinfo.info_data.ingress_qos', tmp)
+                    if tmp2:
+                        from_key = tmp2[0]['from']
```

Comment:
Why not use the same code snipped as in `interface.py`?

```python
for item in tmp2:
    from_key = item['from']
    to_key = item['to']
```

Reply:
done

Reply:
There are other places the old syntax still persists.

Reply:
done